.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+# https://github.com/changesets/action
+name: Release
+
+on:
+  push:
+    branches:
+      - phase21
```

Comment:
Doesn't this need to change as well?

Reply:
First I need to do some mock updates to trigger linked updates. so that is why I need to work on separate branch and keep main branch clean